### PR TITLE
feat : rating and snackbar

### PR DESCRIPTION
**Pull request by someshtiwarii**
Added rating and snackbar component.

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+.snackbar-stacked {
+    max-width: 25rem;
+    display: flex;
+    flex-direction: column;
+    padding: 0.5rem;
+    background-color: var(--dark-grey);
+    color: white;
+    border-radius: 4px;
+}
```
Comment (BelideNikhil) [inline components/snackbar/snackbar.css]: you can create .snackbar as class and write the commonly used properties in all three types

Width, flex, padding, radius, color, background,

**Review by BelideNikhil — APPROVED** [review]
Rest all look good!

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (someshtiwarii) [reply components/snackbar/snackbar.css]: Thankyou Nikhil for pointing it out ! I just did it :)

**Review by srishti-maurya — APPROVED** [review]
looks good

**Pull request merged**